charts/fluent-bit/values.yaml

```diff
@@ -122,6 +122,7 @@ dashboards:
   enabled: false
   labelKey: grafana_dashboard
   annotations: {}
+  # namespace: ""
```

Comment:
```suggestion
  namespace: ""
```

If a value is used by name in a template it should be defined in the values file.

Reply:
Hi @stevehipwell,
thanks for your suggestion! 
I understand, that `with` keyword treats empty string value as false and doesn't render it  (confirmed by running a dry-run).

I based this PR on a similar logic applied already to `prometheusRule`, which also keeps the `namespace` variable commented out. Would you like me to remove the comment from there as well to keep the same standard in both places?